lib/bolt/inventory/inventory.rb

```diff
@@ -209,16 +210,47 @@ def create_target_from_hash(data)
         end
 
         if (aliases = new_target.target_alias)
+          @groups.insert_alia(new_target.name, aliases)
```

Comment:
The validation accepts a String or Array, so this should be `Array(aliases)` to ensure we pass an Array.

Reply:
This variable also seems unnecessary. This seems fine:

```rb
if new_target.target_alias
  @groups.insert_alia(new_target.name, Array(new_target.target_alias))
end
```